wm/client.go

```diff
@@ -0,0 +1,258 @@
+// +build linux,!ci
+
+package wm
+
+import (
+	"github.com/BurntSushi/xgb/xproto"
+	"github.com/BurntSushi/xgbutil/ewmh"
+	"github.com/BurntSushi/xgbutil/icccm"
+	"github.com/BurntSushi/xgbutil/xevent"
+	"github.com/BurntSushi/xgbutil/xprop"
+
+	"fyne.io/desktop"
+	"fyne.io/fyne"
+)
+
+type client struct {
+	id, win xproto.Window
+
+	framed    bool
+	iconic    bool
+	maximized bool
+	title     string
+	class     []string
+	command   string
+	iconName  string
+
+	allowedActions []string
+	stateHints     []string
+
+	fr *frame
+	wm *x11WM
+}
+
+func (s *stack) clientForWin(id xproto.Window) desktop.Window {
```

Comment:
clientForWin returns a Window - seems like a mismatch?

Reply:
I think that was because of testable/non-testable file boundaries, but this is now inside client.go, so returning *client might be fine in this instance?